### PR TITLE
CI: precommit - docformatter

### PR DESCRIPTION
**Pull request by Borda**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?
- [x] Did you write any new necessary tests?

## What does this PR do?
Using https://github.com/myint/docformatter

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by SkafteNicki — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]
LGTM :smiley:

**Comment by Borda** [comment]
seems there is a problem with splitting links to multi-line
```
`Area Under the Receiver Operating Characteristic Curve
     (ROC AUC)<https://en.wikipedia.org/wiki/Receiver_operating_characteristic#Further_interpretations>`_
```
**Inline interpreted text or phrase reference start-string without end-string.**

**Comment by SkafteNicki** [comment]
@Borda since it is only one instance, we can just change that from:
```
`Area Under the Receiver Operating Characteristic Curve (ROC AUC)
    <https://en.wikipedia.org/wiki/Receiver_operating_characteristic#Further_interpretations>`_.
```
to
```
Area Under the Receiver Operating Characteristic Curve (
`ROC AUC <https://en.wikipedia.org/wiki/Receiver_operating_characteristic#Further_interpretations>`_).
```
which hopefully should solve the problem.

**Comment by Borda** [comment]
> since it is only one instance, we can just change that from:

was thinking the same, just wanted first to report it on the package if it is a known issue...

**Comment by Borda** [comment]
asking also here: https://github.com/myint/docformatter/issues/75

File: docs/source/links.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (Borda) [inline docs/source/links.rst:1]: @SkafteNicki resolved by adding central link registry :D

**Pull request merged**